Makefile

```diff
@@ -27,6 +27,7 @@ GOVET_CMD := go tool vet
 CI_HOST_TARGETS ?= "host-unit-test host-integ-test host-build-docker-image"
 SYSTEM_TESTS_TO_RUN ?= "00SSH|Basic|Network|Policy|TestTrigger|ACIM|Netprofile"
 ACI_GW_IMAGE ?= "contiv/aci-gw:04-12-2017.2.2_1n"
+L3_SYSTEM_TESTS_TO_RUN ?= "00SSH|Basic|Bgp|Network|Policy|TestTrigger|ACIM|Netprofile"
```

Comment:
@unclejack, I think we should not filter the L3 tests. Instead we should run all the tests. Last time L3 tests failed because there was a change made that affected the docker service feature. L2 system tests in CI did not catch it as there were filters. Please run all the tests in L3 mode and that can be run once a day (not necessarily for commits)

Reply:
@gkvijay: Thank you for taking a look at this. I'll make sure to check if the tests pass now without filters.